### PR TITLE
DIV-5378 - added managed_identity_object_id

### PR DESCRIPTION
**Pull request by yarpo**
https://tools.hmcts.net/confluence/display/RPE/ASE+to+AKS+migration

based on:
https://github.com/hmcts/service-auth-provider-app/pull/177/files

### CODE REVIEW

File: aat.tfvars [hunk 1]
```diff
@@ -15,3 +15,5 @@ da_external_cert_name = "star-aat"
 external_cert_vault_uri = "https://infra-vault-nonprod.vault.azure.net/"
 
 asp_sku_size = "I3"
+
+managed_identity_object_id = "df18e9f9-5f50-46d8-b75f-34b856fcef24"
```
Comment (qzhou-hmcts) [inline aat.tfvars]: Should this come from the vault?

Comment (hosainnet) [reply aat.tfvars]: Nope, those are just resource IDs that are public

Comment (timja) [reply aat.tfvars]: no it's fine, they aren't credentials

File: perftest.tfvars [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+dn_external_cert_name = "wildcard-perftest-platform-hmcts-net"
+
+da_external_hostname = "decree-absolute.apply-divorce.perftest.platform.hmcts.net"
+
+da_external_cert_name = "wildcard-perftest-platform-hmcts-net"
+
+external_cert_vault_uri = "https://infra-vault-qa.vault.azure.net"
+
+asp_sku_size = "I3"
```
Comment (qzhou-hmcts) [inline perftest.tfvars]: should this prod like environment? on prod we have I2

**Pull request merged**